### ISSUE TITLE
Screen readers do not announce the state of ToolBar ToggleButton

### ISSUE THREAD

**Issue by veselints** [telerik/kendo-ui-core#5391]
### Bug report
Screen readers do not announce the state of the Toolbar ToggleButton

### Reproduction of the problem
1. Open the following demo: https://demos.telerik.com/kendo-ui/toolbar/keyboard-navigation
1. Navigate to the Toggle Button button - its state is not announced.
1. Press enter to change the state of the Toggle Button

### Current behavior
The change in the state is not announced. Also, upon navigating back to that button the state is not announced.

### Expected/desired behavior
The state and the change of the state of the ToggleButton should be announced.

### Environment
* **Kendo UI version:** 2019.3.1023
* **Browser:** all
